server/repositories/cmsApi.js

```diff
@@ -53,7 +53,7 @@ class CmsApi {
     try {
       return await call();
     } catch (err) {
-      if (statusCodes.includes(err.response.status)) {
+      if (err.response && statusCodes.includes(err.response.status)) {
```

Comment:
Do not blow up if we get a timeout (which has no response)

Reply:
You could just use optional chaining.
```if (statusCodes.includes(err?.response?.status)) {```

Reply:
I did and then undid it, just seemed slightly weird doing a check to see if the array contained undefined.. I'll put it back.